packages/aztec.js/src/proof/privateRange/index.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * Constructs AZTEC private range zero-knowledge proofs
+ *
+ * @module privateRange
+ */
+const devUtils = require('@aztec/dev-utils');
+const BN = require('bn.js');
+const { padLeft } = require('web3-utils');
+const crypto = require('crypto');
+
+const bn128 = require('../../bn128');
+const proofUtils = require('../proofUtils');
+const verifier = require('./verifier');
+const Keccak = require('../../keccak');
+
+const privateRange = {};
+privateRange.verifier = verifier;
```

Comment:
It's not clear to me that this is a problem introduced by this pattern. Even if the fn definition for `verifier` happened in the privateRange module, someone with the ability to tamper with the client could simply define a file which requires `privateRange` and redefines `privateRange.verifier`, and as long as that file is loaded prior to the actual use of `privateRange.verifier` in the client, the malicious, replaced fn will be called.

In fact, we use this pattern in tests to replace the method which gets a point from the trusted setup database.

If I misunderstood the attack you were thinking about, could you clarify? Otherwise I think duplicating the `verifier` code is not worth it.

Reply:
As discussed, leaving as is